src/app/shared/components/template/components/layout/form.ts

```diff
@@ -19,23 +19,7 @@ import { Device } from "@capacitor/device";
     </plh-template-component>
     <ion-button (click)="submit()">{{ button_text }}</ion-button>
   </div>`,
-  styles: [
-    `
-      ion-button {
-        font-size: 25px;
-        font-weight: 500;
-        color: var(--ion-color-primary-contrast);
-        text-transform: none;
-        white-space: pre-line;
-        min-height: 71px;
-        padding: var(--regular-margin) var(--regular-padding) 0;
-        --border-radius: 15px;
-        --box-shadow: var(--ion-default-box-shadow);
-        min-width: 100%;
-        --background: var(--ion-btn-primary);
-      }
-    `,
-  ],
+  styleUrls: ["./form.scss"],
```

Comment:
Thanks for tidying this, agree that inline styles get quite messy quite quickly (particularly when we start needing variables)